installer/templates/phx_live/controllers/page_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule <%= web_namespace %>.PageController do
+  use <%= web_namespace %>, :controller
+
+  def index(conn, _params) do
+    Phoenix.LiveView.Controller.live_render(conn, <%= web_namespace %>.PageLiveView, session: %{})
```

Comment:
To simplify this to just be `live_render(...)`, we need to also import the controller helper function

```
  def controller do
    quote do
      ...
      # Import convenience functions for LiveView rendering
      import Phoenix.LiveView.Controller, only: [live_render: 3]

       ...
    end
  end
```

I didn't see this explicitly in the instructions, wasn't sure if that was on purpose or not.

Reply:
We should import it, yes!